akka-operator/templates/040-deployment.yaml

```diff
@@ -31,7 +116,7 @@ spec:
         - name: {{ .Chart.Name }}
           securityContext:
             {{- toYaml .Values.securityContext | nindent 12 }}
-          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+          image: "{{ .Values.defaultAWSRepository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
```

Comment:
Maybe I'm missing something, but are we still capable of overriding the repository used by the deployment?  We need to do this for integration and local testing.

Reply:
Yes, this was also my point. If we hardcode it to `defaultAWSRepository` then how will this value be overridden in case of gcp ? @seglo @andreaTP

Reply:
Yes overridden by the deployer here:
https://github.com/lightbend/akka-cloud-platform-gcp/blob/main/deployer/base-schema.yaml#L42-L52